docs/features/search/concepts.md

```diff
@@ -0,0 +1,88 @@
+---
+id: concepts
+title: Search Concepts
+description: Documentation on Backstage Search Concepts
+---
+
+# Search Concepts
+
+Backstage Search is _blah_.
```

Comment:
```suggestion
Backstage Search is _alpha_.
```
?

Reply:
Haha, this one made me laugh. _It's not that bad at all_

Reply:
As they say...  When you're in the middle of the bell curve, you're really on top.

![md-chart-2x-fs8](https://user-images.githubusercontent.com/3496491/118685245-39426f80-b803-11eb-9394-d07746d39cda.png)